corehq/apps/reports/static/reports/javascripts/reports.config.js

```diff
@@ -43,14 +43,15 @@ var HQReport = function (options) {
                         e.preventDefault();
                         if (self.isExportAll) {
                             $.ajax({
-                                url: get_report_render_url("export"),
+                                url: getReportBaseUrl("export"),
+                                data: getReportParams(undefined, true),
```

Comment:
I think the view receiving this request will see it the same either way, but did you confirm that it still parses the request params correctly?